website/www/site/content/en/contribute/ptransform-style-guide.md

```diff
@@ -24,189 +25,192 @@ _A style guide for writers of new reusable PTransforms._
+- Test that each `withFoo()` method (including each overload) has effect (is not ignored), using `TestPipeline` and `PAssert` to create tests where the expected test results depend on the value of `withFoo()`.
 
 Do not:
-* Do not test successful validation (e.g. "validation does not fail when the transform is configured correctly")
-* Do not test trivial validation errors (e.g. "validation fails when a property is unset/null/empty/negative/...")
+
+- Do not test successful validation (e.g. "validation does not fail when the transform is configured correctly")
+- Do not test trivial validation errors (e.g. "validation fails when a property is unset/null/empty/negative/...")
 
 ### Compatibility
 
 Do:
 
-* Generally, follow the rules of [semantic versioning](https://semver.org/).
-* If the API of the transform is not yet stable, annotate it as `@Experimental` (Java) or `@experimental` ([Python](https://beam.apache.org/releases/pydoc/{{< param release_latest >}}/apache_beam.utils.annotations.html)).
-* If the API deprecated, annotate it as `@Deprecated` (Java) or `@deprecated` ([Python](https://beam.apache.org/releases/pydoc/{{< param release_latest >}}/apache_beam.utils.annotations.html)).
-* Pay attention to the stability and versioning of third-party classes exposed by the transform's API: if they are unstable or improperly versioned (do not obey [semantic versioning](https://semver.org/)), it is better to wrap them in your own classes.
+- Generally, follow the rules of [semantic versioning](https://semver.org/).
+- If the API of the transform is not yet stable, annotate it as `@Experimental` (Java) or `@experimental` ([Python](https://beam.apache.org/releases/pydoc/{{< param release_latest >}}/apache_beam.utils.annotations.html)).
+- If the API deprecated, annotate it as `@Deprecated` (Java) or `@deprecated` ([Python](https://beam.apache.org/releases/pydoc/{{< param release_latest >}}/apache_beam.utils.annotations.html)).
+- Pay attention to the stability and versioning of third-party classes exposed by the transform's API: if they are unstable or improperly versioned (do not obey [semantic versioning](https://semver.org/)), it is better to wrap them in your own classes.
 
 Do not:
 
-* Do not silently change the behavior of the transform, in a way where code will keep compiling but will do something different than the previously documented behavior (e.g. produce different output or expect different input, of course unless the previous output was incorrect).
-Strive to make such incompatible behavior changes cause a compile error (e.g. it's better to introduce a new transform for a new behavior and deprecate and then delete the old one (in a new major version), than to silently change the behavior of an existing transform), or at least a runtime error.
-* If the behavior of the transform stays the same and you're merely changing implementation or API - do not change API of the transform in a way that will make a user's code stop compiling.
```

Comment:
Similarly, there are still a lot of these changes where just bullet points are changed. Please revert these.

I'd suggest just using  `git checkout website-revamp -- website/www/site/content/en/contribute/ptransform-style-guide.md` for each affected file.

Then if it is absolutely necessary to switch from java to language="java" you might use a tool like `sed` to make the changes all at once.

Reply:
yeah I'm using option replace all in vs code